### PR TITLE
Add appIds checking

### PR DESCRIPTION
**Pull request by zdlm**
@wiredearp @zdlm @sampi

Check the appIds

### CODE REVIEW

File: app/dist/ts.app.js [hunk 1]
```diff
@@ -81,7 +83,7 @@ var Listener = function () {
 					return _this;
 				}
 				var content = JSON.parse(e.data.replace(BROADCAST_PREFIX, ''));
-				if (content.key !== key) {
+				if (content.key !== key || content.appIds !== _this.appIds) {
```
Comment (sampi) [inline app/dist/ts.app.js]: you can totally use an arrow `=>` fn here and then you'll have access to `this`

File: app/src/ts.app.js [hunk 2]
```diff
@@ -55,7 +58,7 @@ class Listener {
 				return this;
 			}
 			let content = JSON.parse(e.data.replace(BROADCAST_PREFIX, ''));
-			if (content.key !== key) {
+			if (content.key !== key || content.appIds !== this.appIds) {
```
Comment (sampi) [inline app/src/ts.app.js]: so this is where we should try to use the glob syntax later on, right now we should just have support for `*` or a specific appId

Comment (sampi) [inline app/src/ts.app.js]: or if we really want to go crazy, have `this.appIds` be a type `string` or `array`

File: app/dist/ts.app.js (hunk 1 above)
Comment (zdlm) [inline app/dist/ts.app.js]: This is in the dist folder.

**Pull request merged**